lang/python/python-package.mk

```diff
@@ -121,8 +121,8 @@ PYTHON_PKG_SETUP_ARGS ?= --single-version-externally-managed
 PYTHON_PKG_SETUP_VARS ?=
 
 define PyBuild/Compile/Default
-	$(foreach pkg,$(HOST_PYTHON_PACKAGE_BUILD_DEPENDS),
-		$(call host_python_pip_install_host,$(pkg))
+	$(if $(HOST_PYTHON_PACKAGE_BUILD_DEPENDS),
```

Comment:
it would be a good idea to do `$(strip $(HOST_PYTHON_PACKAGE_BUILD_DEPENDS))`
it looks to me that make's $if() is not that great with some evaluations and you may need to take care with whitespace

Reply:
Hmm the condition is suppose to have [all preceding and trailing whitespace stripped](https://www.gnu.org/software/make/manual/html_node/Conditional-Functions.html) before being expanded and evaluated (whereas the `strip` function should be used for [`ifeq` conditionals](https://www.gnu.org/software/make/manual/html_node/Conditional-Syntax.html))... Are there any examples where it doesn't do it correctly?

Reply:
> Are there any examples where it doesn't do it correctly?

just my paranoia;
this should be fine;
if there are no problems with it, i might use it later in various makefiles i'll write;